### ISSUE TITLE
Switching query does not update the page

### ISSUE THREAD

**Issue by simone-pignotti** [soedinglab/MMseqs2-App#7]
When querying multiple sequences, the queries list on the left of the page redirects to `/i` where `i` is the index of the query sequence.
On chrome, the page URL is updated with the selected index, but the interface freezes until I refresh the page manually.
Maybe the refresh can be automatically triggered when switching query.

**Comment by simone-pignotti** [comment]
I may have detected a related warning in the logs (the requested query had index 3):
```
Jul 10 14:27:28 [my_server] mmseqs[152168]: mmseqs-web-webserver_1  | 2019/07/10 12:27:28 [warn] 6#6: *82 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/3/00/0000000003 while reading upstream, client: [my_IP], server: , request: "GET /api/result/nOToOcKBmH8cYhMmSqN3Fzu7WmB1s0AhFoua3Q/3 HTTP/1.1", upstream: "[docker_address]:3000/result/nOToOcKBmH8cYhMmSqN3Fzu7WmB1s0AhFoua3Q/3", host: "[my_server_address]:8877", referrer: "[my_server_address]:8877/result/nOToOcKBmH8cYhMmSqN3Fzu7WmB1s0AhFoua3Q/3"
```

**Comment by milot-mirdita** [comment]
I can't reproduce the issue here.
Can you post the query sequences and what database you search against? Does this happen against any large database (e.g. Uniclust)?

**Comment by simone-pignotti** [comment]
Unfortunately I can't post the sequences, but I realized it is only happening when querying a nucleotide database with `--search-type 2` (proteins VS translated ORFs). This is the exact params used:
```
{"status":"COMPLETE","display":{"name":"all_collections_aa2nt","version":"2019-07-10","path":"all_collections_aa2nt","default":true,"order":0,"index":"-s 7.5 --translation-table 11 --use-all-table-starts true --search-type 2","search":"-s 7.5 --search-type 2"}}
```
Querying multiple proteins to such a nucleotide index triggers the issue. Sorry for not including this essential information!

**Comment by milot-mirdita** [comment]
Could you try the latest docker once it is done building. Change the `TAG` in `docker-compose/.env` to `latest` and it should use the latest docker. 

I couldn't reproduce the problem locally yet, but I fixed some other issues with empty results (both in MMseqs2 and in the app).

**Comment by simone-pignotti** [comment]
Hey, no unfortunately it does not seem to be solved with the last build!

**Comment by simone-pignotti** [comment]
BTW I would say this is low priority, as a manual page refresh works fine

**Comment by milot-mirdita** [comment]
I tried reproducing the problem a few days ago again. Double checking that I am using Chrome, the correct search mode, etc. I couldn't get it to break.

If its not possible to send me the sequences where it breaks, would it be possible to give me access to the URL of the server somehow? So I can step through the web interface in a debugger and try to find out whats going wrong. You could send me the URL or data in private (twitter or email).

**Comment by simone-pignotti** [comment]
Hi,

I had a hard time reproducing this issue myself :) I was trying to build a toy DB and query fasta, but with small datasets everything works well. Therefore I have built a DB including all E. coli genomes from RefSeq, and downloaded the first 1k proteins from https://www.uniprot.org/uniref/UniRef50_P00722 to use as queries. You will find all the data to reproduce it [here](https://drive.google.com/file/d/1XRQ1XoytAWzqGjEu5XqumuEYiQ2tGTF4/view?usp=sharing) (index was quite big so I didn't upload it, but I am using the latest docker version [4] so you should be able to build the exact same index from the params file).

I experience the issue somehow randomly when switching query: the same query may hang the web page, but after reloading and switching to it again works fine. You can try to switch multiple queries rapidly to increase the chances of getting it to hang.

Let me know if you manage to observe that. Unfortunately I cannot send you the data because of the company policies, and the server is only accessible from the company's LAN.

**Comment by milot-mirdita** [comment]
I was at a conference and couldn't update earlier. Thanks a lot, I was able to reproduce the issue with your example, however didn't have time to really find the root cause yet. That was very helpful though!

**Comment by simone-pignotti** [comment]
No problem, glad that helped!

**Comment by milot-mirdita** [comment]
I think this issue should be fixed since a while now. I think it was caused by the short timeouts in the go http server. They are now long gone.
